tests/test_cases/fork_simple.py

```diff
@@ -0,0 +1,34 @@
+#!/bin/python2
+# Both fork_simple.c and template.c work similarly, the main common aspect being both of them have different ordered forks with conflicting keys.
+# A simple solution that covers both of them would be sorting the results, and comparing the lines for deterministic inputs, while checking the pid's for errors.
+
+import re
+import sys
+import string
+
+def template(results):
+    pidlist = re.findall('[-\d]+', results)
+    print pidlist
+
+lind_results = sys.argv[1]
+native_results = sys.argv[2]
+
+template(lind_results)
+template(native_results)
```

Comment:
The template stuff doesn't actually process anything, just prints things out, right? If so is it necessary?

Reply:
Not actually, it's there to help browsing the output. Removed.